test/config/project-config_test.js

```diff
@@ -34,15 +51,9 @@ suite('Project Config', () => {
     assert.deepEqual(config.fragments, [path.resolve('foo.html')]);
   });
 
-  test('inits from config file', () => {
```

Comment:
It's equivalent (after refactor) is now 'reads options from config file' https://github.com/Polymer/polymer-cli/pull/231/files/156f4a52154e51904c973fcf4627e749e9a73c98#diff-062b6039a01f744d6481ace6873ad64eR18